.github/workflows/codex-feedback-archive-relay.yml

```
@@ -0,0 +1,205 @@
+
+permissions:
+  actions: write
+  checks: write
+  contents: read
+  issues: write
+  pull-requests: read
+
+jobs:
+  relay:
+    name: Persist fork feedback archive and publish gate
+    if: startsWith(github.event.workflow_run.display_title, 'Codex P1 Gate relay-v1 PR #')
```

actionlint (1.7.12)

[error] got unexpected EOF while lexing end of string literal, expecting '''

(expression)